src/time/gadgets.h

```diff
@@ -98,29 +147,4 @@ operator ==(const timeval &a, const timeval &b)
     return !(a != b);
 }
 
-/// prints <seconds>.<microseconds>
-std::ostream &operator <<(std::ostream &, const timeval &);
-
-namespace Time
-{
-
-/** Display time as a formatted human-readable string.
- * Time syntax is
- * "YYYY/MM/DD hh:mm:ss"
- *
- * Output is only valid until next call to this function.
- */
-const char *FormatStrf(time_t t);
-
-/** Display time as a formatted human-readable string.
- * Time string syntax used is that of Apache httpd.
- * "DD/MMM/YYYY:hh:mm:ss zzzz"
- *
- * Output is only valid until next call to this function.
- */
-const char *FormatHttpd(time_t t);
-
-} // namespace Time
```

Comment:
If possible, please return the deleted/moved lines above to their original place to expose the actual changes (if any).

Reply:
The change *is* the move itself to group the parts of this squid_curtime API together.

Reply:
Done in [82347a0](https://github.com/squid-cache/squid/pull/1001/commits/82347a06be8f3cd9f24ab99432d7713d3fd8c87e)